lib/Mojolicious/Renderer.pm

```diff
@@ -107,7 +107,14 @@ sub render {
 
   # Inheritance
   my $content = $stash->{'mojo.content'} //= {};
-  local $content->{content} = $output =~ /\S/ ? $output : undef if $stash->{extends} || $stash->{layout};
+  local $content->{content} = $output =~ /\S/ ? $output : undef
+    if $stash->{extends} || $stash->{layout} || $stash->{inline_layout};
+  if (defined $stash->{inline_layout}) {
+    local $options->{inline} = delete $stash->{inline_layout};
+    delete $stash->{layout};
```

Comment:
Do the tests still pass if this line is removed?

Reply:
They do. I tried to write something to test this line but did not succeed (setting `layout` and `inline_layout` together in a call to render results in an error 500 but I’m not sure why). I still have the impression that this line should stay.

Do you have an idea of how to test it? Or do you think that it should be removed?

Reply:
Code that does nothing should not be present.